docs/build-the-docs.md

```diff
@@ -24,7 +24,8 @@ docker run -ti --rm \
       --cache-dir /antora/cache/ \
       --generator=./generate-site.js \
       --stacktrace \
-        site.prod.yml
+      --url <URL or path to your Antora instance> \
```

Comment:
I would write here:
``your Antora instance`` --> ``your Antora docs repro instance``
It may be that thre are more than one documentation repositories for other projects and this makes it more clear - imho.

Reply:
That should be: URL for the documentation

Reply:
I just looked at the antora documentation, https://docs.antora.org/antora/1.1/cli/
``--url <url>``
```
Base URL of the published site.
The URL should not include a trailing slash.
Overrides the site url value specified in the playbook
```
Means for me,
the playbook should contain as ``<url>`` --> ``https://docs.owncloud.org``
and should be overwritten when you build the docs by the url your local webserver uses to access the built documentation as this is for maintaining it.

Reply:
This attribute defines the base URL, right. That's why this always have to be some local webserver URL. For production, this is anyway totally automated by drone.

Reply:
Thanks for the final point of clarity, @tboerger.